src/components/v2/Select/index.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { ComponentMeta } from '@storybook/react';
+import { withCenterStory } from 'stories/decorators';
+import { Select } from '.';
+
+export default {
+  title: 'Components/Select',
+  component: Select,
+  decorators: [withCenterStory({ width: 200 })],
+  parameters: {
+    backgrounds: {
+      default: '#1F2028',
```

Comment:
Ideally this value comes from the enum of options we have set `Primary` `Paper` `White`. They are set in `.storybook/preview.js` if not we can grab them from the Palette constant